Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 22, in main
    dataset_train, dataset_val = build_dataset(args.clustering_framework, args.dataset, args.dataset_path, args)
AttributeError: 'Namespace' object has no attribute 'clustering_framework'
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    main(args, logger)
  File "main.py", line 16, in main
    torch.cuda.set_device(args.gpu)
AttributeError: 'Namespace' object has no attribute 'gpu'
